### PR TITLE
Modify Message

### PR DESCRIPTION
**Pull request by khaled-11**
More Clear and easy explain of the Open-Source

#### Is your pull request related to a problem? Please describe
No just change of definition in conversation text.

#### Describe the solution you'd like
I replaced the text in the Four Locales

#### Test plan
A good test plan should give instructions that someone else can easily follow.
Also add a link to your own Messenger experience for your reviewer to test the changes.

#### Checklist
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **[CONTRIBUTING](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.
- [ ] I have added the label **enhancement** to my pull request.

### CODE REVIEW

**Comment by aboullaite** [comment]
Hello @khaled-11! Can you please highlights how this change is better than the current version ?

**Comment by khaled-11** [comment]
This was the original version: 
"According to the dictionary, Open-source 🔓 software, denotes software for"
" which the original source code is made freely 🆓 available and may be "
"redistributed and modified."

** I felt it more like Google Translate and didn't make sense and weren't clear enough for me. I had to read it multiple times to get the meaning.


This is the updated version: 
"Open-Source Software, is any Software available for modification ⌨️."
"This usually includes a license for programmers to change and contribute to the software. 🔧"
"Programmers can fix bugs, improve the software, or adapt it according to their own needs! 🆓"

** I feel this more clear and it quickly deliver the meaning.

File: src/locales/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -122,9 +122,9 @@ msgstr ""
 
 #: services/messenger.py:236
 msgid ""
-"According to the dictionary, Open-source 🔓 software, denotes software for"
-" which the original source code is made freely 🆓 available and may be "
-"redistributed and modified."
+"Open-Source Software, is any Software available for modification ⌨️."
```
Comment (elinguiuriel) [inline src/locales/fr/LC_MESSAGES/messages.po:125]: Is this an authoritative definition? It seems to belong to Patricia Picket: https://www.thebalancecareers.com/what-is-open-source-software-2071941

A much formal definition can be  **Open source software is software with source code that anyone can inspect, modify, and enhance.** : https://opensource.com/resources/what-open-source , which for me is quite similar to the version you want to modify!

Comment (khaled-11) [reply src/locales/fr/LC_MESSAGES/messages.po:125]: Yeah. This definition is similar to the one in Patricia Picket article.
I feel the Opensource.com definition sounds better. Do you want to update it, or should I update it?

**Comment by Awinja-j** [comment]
Hey @khaled-11 ,

Thanks for the contribution. This PR is ripe for merging. Please solve the conflicts so that we can have it merged soon.

**Comment by Awinja-j** [comment]
Thank you so much @khaled-11 . Good job

**Pull request merged**